### PR TITLE
UI improvments

### PR DESCRIPTION
**Pull request by dark-eye**
This is a general clean up and UX improvements to the  clock :
**Note This is based on : https://github.com/ubports/clock-app/pull/4**
- Made the bottom edge  ( from  https://github.com/ubports/clock-app/pull/4 ) match better with  Material Design. 
- Decreased scroll hangs on stopwatch lap list (when the item consume the swipe before the list) by adding pressDelay to the list
-Decreased scroll hangs on Alarms list (when the item consume the swipe before the list) by adding pressDelay to the list
- cleaned up the swiping code a little.
- fixed an issue that caused the bottom Navigation  to mark the  wrong page as the   current page.

**New Look :**
- Clock 
image

- Stopwatch :
image

### CODE REVIEW

**Comment by mariogrip** [comment]
@Flohack74 want to take a look?

File: app/MainPage.qml [hunk 1]
```diff
@@ -20,6 +20,8 @@ import QtQuick 2.4
 import Ubuntu.Components 1.3
 import QtSystemInfo 5.0
 import Qt.labs.settings 1.0
+import QtGraphicalEffects 1.0
```
Comment (dark-eye) [inline app/MainPage.qml]: This was committed by mistake i`ll remove it tomorrow

**Comment by Flohack74** [comment]
No its ok I think, lets see how it works on the phone.

**Comment by dark-eye** [comment]
No it shouls be ok but i suggest to wait just to make the merge cleaner ill push a new commit in a few minutes

**Pull request merged**